backend/migrations/0001_create_message_table.down.sql

```diff
@@ -1,2 +1,2 @@
--- Add down migration script here
+-- create_users_table.down
```

Comment:
```suggestion
-- Add down migration script here
```

Migrations shouldn't be changed, it will probably break the checksum and cause issues

Reply:
Checksums might ignore comments, but this comment change is incorrect anyways, so just revert it